qcodes/dataset/descriptions.py

```diff
@@ -0,0 +1,67 @@
+import io
+from typing import Dict, Any
+
+from ruamel.yaml import YAML
+
+from qcodes.dataset.dependencies import InterDependencies
+
+
+class RunDescriber:
+    """
+    The object that holds the description of each run in the database. This
+    object serialises itself to a string and is found under the run_description
```

Comment:
"...serializes itself..." - is `serialize` or `to_yaml` referred to here?

also, the docstring of the class does not have to state where the object is going to be "dumped" to with this precision :)

Reply:
> "...serializes itself..." - is serialize or to_yaml referred to here?

The glorious combination of the two.